### PR TITLE
Account dashboard mvp

### PR DESCRIPTION
**Pull request by tomquirk**
Screenshot 2023-02-23 at 2 53 32 PM
Screenshot 2023-02-23 at 2 53 39 PM
Screenshot 2023-02-23 at 2 53 42 PM

### CODE REVIEW

File: src/pages/address/[address]/index.page.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+import { AppWrapper } from 'components/common'
+import { useRouter } from 'next/router'
+
+export default function AddressPage() {
+  const router = useRouter()
+  const { address } = router.query
+
+  if (!address) return null
+
+  return (
```
Comment (wraeth-eth) [inline src/pages/address/[address]/index.page.tsx:11]: ❤️

File: src/pages/address/[address]/index.page.tsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
+import { AddressDashboard } from 'components/AddressDashboard'
+import { AppWrapper } from 'components/common'
+import { useRouter } from 'next/router'
+
+export default function AddressPage() {
+  const router = useRouter()
+  const { address } = router.query
+
+  if (!address) return null
```
Comment (wraeth-eth) [inline src/pages/address/[address]/index.page.tsx:9]: 404 not found

File: src/pages/address/[address]/index.page.tsx [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import { AddressDashboard } from 'components/AddressDashboard'
```
Comment (wraeth-eth) [inline src/pages/address/[address]/index.page.tsx:1]: wdyt of `account/` as route, then we can use same route for ens one day too

**Review by wraeth-eth — CHANGES_REQUESTED** [review]

File: src/pages/address/[address]/index.page.tsx (hunk 3 above)
Comment (tomquirk) [reply src/pages/address/[address]/index.page.tsx]: done

File: src/pages/address/[address]/index.page.tsx (hunk 2 above)
Comment (tomquirk) [reply src/pages/address/[address]/index.page.tsx]: done via getServerSideProps, i think thats the only way? lemme know

**Pull request merged**